composer.json

```diff
@@ -27,7 +26,7 @@
         }
     },
     "require-dev": {
-        "orchestra/testbench": "3.2.*"
+        "phpunit/phpunit": "^6.0"
```

Comment:
You should be ok to add an or for versions higher than 6. That way higher php versions can get an up to date phpunit install: `^6.0 || ^7.0`

Reply:
Updated.